src/views/admin/RolleCreationView.vue

```diff
@@ -253,305 +422,8 @@
-                  cols="3"
-                  class="d-none d-md-flex"
-                ></v-col>
-                <v-col
-                  cols="12"
-                  md="3"
-                  class="md-text-right py-0"
-                >
-                  <label
-                    class="text-body"
-                    for="rollenname-input"
-                    required="true"
-                  >
-                    {{ $t('admin.rolle.rollenname') }}
-                  </label></v-col
-                >
-                <v-col
-                  cols="12"
-                  md="auto"
-                  class="py-0"
-                >
-                  <v-text-field
-                    clearable
-                    data-testid="rollenname-input"
-                    id="rollenname-input"
-                    v-model="selectedRollenName"
-                    :placeholder="$t('admin.rolle.enterRollenname')"
-                    variant="outlined"
-                    density="compact"
-                    :rules="rolleNameRules"
-                    required
-                  ></v-text-field>
-                </v-col>
-              </v-row>
-              <!-- select characteristics -->
-              <v-row>
-                <v-col
-                  class="d-none d-md-flex"
-                  cols="2"
-                ></v-col>
-                <v-col>
-                  <label class="subtitle-2">4. {{ $t('admin.rolle.assignMerkmale') }}</label>
-                </v-col>
-              </v-row>
-              <v-row>
-                <!-- Spacer column -->
-                <v-col
-                  cols="3"
-                  class="d-none d-md-flex"
-                ></v-col>
-                <v-col
-                  cols="12"
-                  md="3"
-                  class="md-text-right py-0"
-                >
-                  <label
-                    class="text-body"
-                    for="merkmale-select"
-                  >
-                    {{ $t('admin.rolle.merkmale') }}
-                  </label></v-col
-                >
-                <v-col
-                  cols="12"
-                  md="auto"
-                  class="py-0"
-                >
-                  <v-select
-                    data-testid="merkmale-select"
-                    id="merkmale-select"
-                    :items="translatedMerkmale"
-                    v-model="selectedMerkmale"
-                    item-value="value"
-                    item-text="title"
-                    :placeholder="$t('admin.rolle.selectMerkmale')"
-                    variant="outlined"
-                    density="compact"
-                    multiple
-                    clearable
-                  >
-                    <template v-slot:item="{ props, item }">
-                      <v-list-item
-                        v-bind="props"
-                        :title="item.title"
-                        class="select-item-text"
```

Comment:
Kann man das irgendwie in die Form integrieren? Die Texts im Dropdown sind jetzt leider Schwarz.

Reply:
Was genau ist schwarz? Bei mir steht
![image](https://github.com/dBildungsplattform/schulportal-client/assets/141040785/016c45c6-4e86-4636-a947-0b6bd31c9059)

Reply:
fixed via vuetify variables